web/impact/impact/v1/views/impact_view.py

```diff
@@ -1,48 +1,68 @@
 from abc import (
     ABCMeta,
-    abstractmethod,
+    # abstractmethod,
```

Comment:
Should be removed if unused.

Reply:
and if so, the parenthesis should be removed as well.

Reply:
Done.